roles/webots/localisation.role

```diff
@@ -22,10 +22,8 @@ nuclear_role(
   # Vision
   vision::VisualMesh
   vision::GreenHorizonDetector
-  vision::GoalDetector
-  vision::BallDetector
-  vision::RobotDetector
   vision::FieldLineDetector
+  vision::Yolo
```

Comment:
Does yolo run faster on cpu in webots? I'm not sure of the benefit to removing the visual mesh objects, for development purposes. it would be good to be able to distinguish which model detected what object.

Reply:
If you want to use GPU you have to give docker access to GPU which is a little tedious so best to leave as CPU